### ISSUE TITLE
Saving model on drive when training on Colab

### ISSUE THREAD

**Issue by nalin-mathur** [tensorflow/ranking#106]
I have been using Google Colab to train my ranking model using the 'example/tf_ranking_libsvm.ipynb' and suitably making changes to it for using in Colab. I plugged in my google drive into the notebook and have been using directories in my drive as the '--train_path', '--vali_path', '--test_path', and '--output_dir'. 

The model is training as expected and the loss is going down. The model also gets saved into drive as checkpoints into the '--output_dir'. 
However on the last step of training, when saving the final model an exception occurs which I cannot fully understand. I want to use the final model to make predictions but it isn't being saved into drive due this exception. Can someone please help me with figuring out the reason for this exception?
![image](https://user-images.githubusercontent.com/25167197/63411674-aa2cdb80-c413-11e9-8c1d-ea53f644c61c.png)

**Comment by nalin-mathur** [comment]
I was running another script on colab which was not training a model or saving it onto drive. The same exception occurred and it again traces back to 'sys.exit(main(argv))'. 

Hence, I believe this has something to do with Colab and not tensorflow_rank.

**Comment by bendersky** [comment]
Nalin, this does not seem to be an issue related to TF-Ranking but rather with your Colab setting, so I am closing this for now -- please let us know any library related issues. Thanks!
